pkg/apis/minio.min.io/v2/types.go

```diff
@@ -360,6 +360,17 @@ type AuditConfig struct {
 
 // PrometheusConfig defines configuration for Prometheus metrics server
 type PrometheusConfig struct {
+	// Image defines Prometheus container image.
+	// +optional
+	Image string `json:"image,omitempty"`
+	// SideCarImage defines container image to be used for sidecar
+	// Expects "alpine" image
+	// +optional
+	SideCarImage string `json:"sidecarimage,omitempty"`
```

Comment:
I think sidecars should be an array of Container's, similar to what we did for zones, that way if they need more sidecars it's easier to add